scripts/build-webpack.js

```diff
@@ -33,7 +36,7 @@ const compiler = webpack({
     }),
   ],
   output: {
-    filename: `yarn-${version}.js`,
+    filename: `[name]-${version}.js`,
```

Comment:
Hmmmm. I wonder if we should version the lockfile parser separately. We could call the current one `1.0`, and bump it to `2.0` if there's any major changes. What do you think?